test/e2e/main_test.go

```diff
@@ -453,24 +454,20 @@ func TestExternalService(t *testing.T) {
 	}
 }
 
-func exposeServiceSpec() corev1.ServiceSpec {
-	return corev1.ServiceSpec{
-		Type:  exposeServiceType(),
-		Ports: []corev1.ServicePort{{NodePort: 30222}},
+func exposeServiceSpec() ispnv1.ExposeSpec {
+	return ispnv1.ExposeSpec{
+		Type:     exposeServiceType(),
+		NodePort: 30222,
 	}
 }
 
-func exposeServiceType() corev1.ServiceType {
-	exposeServiceType := tconst.ExposeServiceType
+func exposeServiceType() ispnv1.ExposeType {
```

Comment:
@rigazilla have you plan to add specific OCP route test, which will be skipped on k8s running?

Reply:
I would like to track this on a separate issue

Reply:
Yes, sure, separate test from core dev PR is prefer way

Reply:
https://github.com/infinispan/infinispan-operator/issues/410